cmd/main.go

```diff
@@ -102,6 +104,23 @@ func StartNSXServiceAccountController(mgr ctrl.Manager, commonService common.Ser
 	}
 }
 
+func StartSubnetPortController(mgr ctrl.Manager, commonService common.Service) {
```

Comment:
Looks like every controller has a similar way to start, is it doable to have a generic way for all of them?

Reply:
Addressed https://github.com/vmware-tanzu/nsx-operator/pull/165/files#r1279020987.